src/data.js

```diff
@@ -2087,6 +2087,17 @@ module.exports = [
       'Vue',
     ],
   },
+  {
+    name: 'David Petringa',
+    description:'Web developer coding in PHP, Laravel, Vuejs. Blogger as a side Hustle.',
+    url: 'https://david.dukesnus.com/',
```

Comment:
Should be a /uses URL. This page is a 404. Please fix

Reply:
So sorry, Fixed now. and I resubmitted pull request.